.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -1,22 +1,34 @@
-Hi there,
+---
+name: Bug Report
+about: If something isn't working as expected.
 
-Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.
+---
+
+Hello,
+
+Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved
+for bug reports and feature requests. For general usage questions, please see:
+https://www.terraform.io/community.html.
 
 ### Terraform Version
 Run `terraform -v` to show the version. If you are not running the latest version of Terraform, please upgrade because your issue may have already been fixed.
 
 ### Affected Resource(s)
 Please list the resources as a list, for example:
-- opc_instance
-- opc_storage_volume
+- vcd_vapp_vm
+- vcd_network_routed_v2
 
 If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.
 
 ### Terraform Configuration Files
+
 ```hcl
-# Copy-paste your Terraform configurations here - for large Terraform configs,
-# please use a service like Dropbox and share a link to the ZIP file. For
-# security, you can also encrypt the files using our GPG public key.
+Copy-paste your Terraform configurations here. Please **shrink HCL replicating the bug to minimum** 
+without modules. It will help us to respond and replicate the problem quicker. It may also uncover
+some HCL error while doing so.
+
+For bigger configs you may use [Gist(s)](https://gist.github.com) or a service like Dropbox and
+share a link to the ZIP file. For security, you can also encrypt the files using our GPG public key.
```

Comment:
I have actually left this sentence because it was there, but as we have never used it and GPG key is not hosted - I am removing this altogether for now.